src/ast.rs

```diff
@@ -44,6 +44,8 @@ impl Debug for POU {
 pub struct Implementation {
     pub name: String,
     pub type_name: String,
+    pub linkage : LinkageType,
```

Comment:
does this mean that the POU's linkage could be removed?

Reply:
Yes, i think so i'll try to do it next.